Clarify configurable back pressure properties (#1515)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-shadow/rpk-shadow-describe.adoc

By default, the command displays the overview and client configuration sections. Use the flags to display additional sections such as topic synchronization, consumer offset synchronization, and security synchronization settings.

> For Redpanda Cloud, `rpk` will use the Redpanda ID of the cluster you are currently logged into. If you wish to use a different one either login and create a profile for it, or use the `--redpanda-id` flag to specify it directly.

Reviewer: ```suggestion
For Redpanda Cloud, `rpk` will use the Redpanda ID of the cluster you are currently logged into. To use a different cluster, either log in and create a profile for it, or use the `--redpanda-id` flag to specify it directly.
```
